iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -2112,4 +2124,12 @@ public List<FileStatus> getMergeTaskInputFiles(Properties properties) throws IOE
   public MergeTaskProperties getMergeTaskProperties(Properties properties) {
     return new IcebergMergeTaskProperties(properties);
   }
+
+  @Override
+  public void setCustomDeleteProperties(TableDesc tableDesc) {
+    tableDesc.getProperties().put(InputFormatConfig.OPERATION_TYPE_PREFIX + tableDesc.getTableName(),
+            Operation.DELETE.name());
+    tableDesc.getProperties().put(HiveCustomStorageHandlerUtils.WRITE_OPERATION_CONFIG_PREFIX +
```

Comment:
why not use API
````
HiveCustomStorageHandlerUtils.setWriteOperation(tableDesc.getProperties(), tableDesc.getTableName()Operation.DELETE)
````

Reply:
Added util methods in TableDesc to get and set property.